packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -541,5 +541,7 @@ export const messages = {
     'Package ID should be a 15 or 18 character Id that starts with 04t',
   package_id_gatherer_placeholder: '04t...',
   force_function_enter_function: 'Enter a name for the function',
-  force_function_enter_language: 'Select a language for your function'
+  force_function_enter_language: 'Select a language for your function',
+  force_function_pull_dependencies_error: "Error pulling dependencies. Try running 'npm install' for the function yourself",
```

Comment:
@jag-sfdc - thinking this error could occur because npm isn't installed or no dependencies are defined in package.json. If so, would like to present both. cc: @danyusf

Reply:
Will the error info be captured in the Output pane in VSCode? That should help determine the cause of the failure for most devs (e.g. "npm: command not found")

Note that npm is a requirement for developing Node functions (although this may or may not be true for the upcoming "Apex functions", which isn't in this extension's scope).

Reply:
Yes, there is a way to surface the error (like "npm: command not found"). I'll make that change. So, can I change the text to "Error pulling dependencies - <actual error text here>" ?

Reply:
@ekapner - This how the error message would look now
![Screen Shot 2020-09-03 at 9 24 33 AM](https://user-images.githubusercontent.com/58611665/92143123-a9ba7500-edc9-11ea-81f7-c48ab6b8cd96.png)

Reply:
What do you think, @danyusf? Sounds like we may swap out "pull" for "install" - based on feedback from Luis.

Reply:
@ekapner yes, let's use install